runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -73,6 +73,14 @@ default boolean showHAValue()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "includeNatureCost",
+			name = "Include Nature Rune Cost",
+			description = "Configures whether or not to display item HA value after the cost of nature runes",
+			position = 4
```

Comment:
Too many tabs (it should be 1 tab instead of 2, compare with the other settings)